numpy/lib/financial.py

```diff
@@ -597,6 +605,8 @@ def irr(values):
     that gives a net present value of 0.0; for a more complete explanation,
     see Notes below.
 
+    Decimal type is not supported for values.
```

Comment:
I wonder if `` `~decimal.Decimal` `` would link to the python docs here. @MSeifert04? - you seem to have a bunch of experience with Sphinx